src/main/java/parser/Parser.java

```diff
@@ -290,6 +295,79 @@ private String removeViewPrefix(String userInput) {
         return userInput.replace(VIEW_COMMAND, EMPTY_REGEX);
     }
 
+    //@@author iaso1774
+    /**
+     * Processes the user input to change the deadline for a Manga.
+     * Determines whether the input corresponds to an existing Author and Manga.
+     * Converts the inputted date to LocalDate type if possible.
+     * <p>
+     * If the input is valid, it returns an `AddDeadlineCommand`.
+     *
+     * @param userInput the input string provided by the user,
+     *                  which should specify a Schedule operation for a specified Manga and Author
+     * @return an AddDeadlineCommand object
+     * @throws TantouException if the user input is invalid
+     */
+    private Command processScheduleCommand(String userInput) throws TantouException {
+        String authorName = null;
+        String mangaName = null;
+        String deadline = null;
+
+        AuthorArgumentFinder authorArgumentFinder = new AuthorArgumentFinder();
+        MangaArgumentFinder mangaArgumentFinder = new MangaArgumentFinder();
+        DeadlineArgumentFinder deadlineArgumentFinder = new DeadlineArgumentFinder();
+
+        ArgumentResult authorResult = authorArgumentFinder.getArgumentResult(userInput);
+        authorName = authorResult.getArgument();
+        String userInputPostAuthorExtraction = authorResult.getOutputString();
+
+        if (authorName == null || authorName.isEmpty()) {
+            throw new NoAuthorProvidedException();
+        }
+
+        ArgumentResult mangaResult = mangaArgumentFinder.getArgumentResult(userInputPostAuthorExtraction);
+        mangaName = mangaResult.getArgument();
+
+        if (mangaName == null || mangaName.isEmpty()) {
+            throw new NoMangaProvidedException();
+        }
+
+        ArgumentResult deadlineResult = deadlineArgumentFinder.getArgumentResult(userInputPostAuthorExtraction);
+        deadline = deadlineResult.getArgument();
```

Comment:
You may want to use `userInputPostMangaExtraction` here instead! This is because it allows for all the arguments processed before `deadlineResult` to be omitted.